src/libs/SoloX.BlazorJsonLocalization/HttpHostedJsonLocalizationOptions.cs

```diff
@@ -26,6 +26,11 @@ public class HttpHostedJsonLocalizationOptions : AJsonExtensionOptions
         /// </summary>
         public string CultureSeparator { get; set; } = "-";
 
+        /// <summary>
+        /// Adds a querystring version to the resource path. Use it for cache busting
+        /// </summary>
+        public string Version { get; set; } = string.Empty;
```

Comment:
How about using the assembly version?
In that case we would have an options like EnableAssemblyVersionSuffix because if you use shared component with localization files, a global version suffix would not necessary match all components versions.
What do you think?

Reply:
or may be we could consider using an attribute on the class itself like ResourceSuffixAttribute:
```csharp
namespace SoloX.BlazorJsonLocalisation.Example.Components.StaticAssets.Pages
{
    [ResourceSuffix("-v2")]
    public partial class Counter
    {
        [Inject]
        private IStringLocalizer<Counter> L { get; set; }

        /* ... */
    }
}
```

giving use the resource file name like:

Counter-v2-en.json

This solution would give us more control.

Reply:
My initial idea was something like what you first suggested (using the assembly version).
I thought of using it as a global version in the options mostly because that's how I've been controlling the version of my apps.

It's true that using an attribute in each component would make it more flexible and avoid busting the cache of every resource in the project even when only one (or even none) was changed. But it would also require manual intervention of each resource every time a simple change was made.

I'd suggest that both options would be useful. Each one being optional

Reply:
Ok let's implement the global version first. I'will do the more customizable later if the needs requires it....

Reply:
Your NamingPolicy suggestion kind of makes it as flexible as we could need
I don't have the time today to actually implement this but I should give it a look tomorrow!